server/svix-server/src/v1/endpoints/message.rs

```diff
@@ -198,93 +193,36 @@ pub struct CreateMessageQueryParams {
 async fn create_message(
     Extension(ref db): Extension<DatabaseConnection>,
     Extension(queue_tx): Extension<TaskQueueProducer>,
-    Extension(cache): Extension<Cache>,
     ValidatedQuery(CreateMessageQueryParams { with_content }): ValidatedQuery<
         CreateMessageQueryParams,
     >,
     ValidatedJson(data): ValidatedJson<MessageIn>,
     AuthenticatedApplication { permissions, app }: AuthenticatedApplication,
 ) -> Result<(StatusCode, Json<MessageOut>)> {
-    let create_message_app = CreateMessageApp::layered_fetch(
```

Comment:
You do want it and need it. Don't remove it, just don't create messagedestinations from it.

It's used for error handling and to know whether there's even a need to create a task batch (what if there are no endpoints that match this message? E.g. due to event type filtering).